docs/references/strategies/README.md

```diff
@@ -145,40 +145,49 @@ Invoke strict analysis with the `--strict` flag when running `fossa analyze`.
-| [Python (Pdm)](./languages/python/pdm.md)                                                                                                       | ❌         | ✅         | ❌                    | Static           |
-| [Python (setup.py/requirements.txt)](https://github.com/fossas/fossa-cli/blob/master/docs/references/strategies/languages/python/setuptools.md) | ✅         | ✅         | ❌                    | Dynamic          |
-| [R (renv)](./languages/r/renv.md)                                                                                                               | ❌         | ✅         | ❌                    | Static           |
-| [Ruby (bundler)](https://github.com/fossas/fossa-cli/blob/master/docs/references/strategies/languages/ruby/ruby.md)                             | ✅         | ✅         | ❌                    | Static           |
-| [Rust (cargo)](https://github.com/fossas/fossa-cli/blob/master/docs/references/strategies/languages/rust/rust.md)                               | ✅         | ❌         | ❌                    | Dynamic          |
-| [Scala (sbt)](https://github.com/fossas/fossa-cli/tree/master/docs/references/strategies/languages/scala)                                       | ✅         | ❌         | ❌                    | Dynamic          |
-
-:warning:: Note that these strategies support _static_ and _dynamic_ detection differently than other strategies, and are not run by default.
-   Please make sure to check their linked documentation in the table above for more details.
+> [!NOTE]
+> Dynamic strategies require a working build environment for analysis.
+>
+> If a given package manager has a dynamic strategy with a static fallback, that means the static fallback provides worse results,
+> so it is only used when the dynamic strategy fails. If the package manager only has static strategies, that means dynamic analysis
+> is not required for ideal results.
+
+> [!TIP]
+> If FOSSA CLI is forced to utilize a fallback strategy, meaning it did not detect ideal results,
+> a warning is emitted in the scan summary after running `fossa analyze`.
+
+> [!WARNING]
+> "Custom" strategies work very differently than the standard package manager based analysis; read their docs for more details.
+
+| Language/Package Manager                                                                                                                        | Kind of analysis             | Detect Vendored Code |
+|-------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------|----------------------|
+| [C#](https://github.com/fossas/fossa-cli/tree/master/docs/references/strategies/languages/dotnet)                                               | Static                       | ❌                    |
```

Comment:
[Question] Is Nuget always what's used for C#? I don't need this resolved for approval but it feels like the package manager or NuGet is the right thing to put here vs. the language.

Reply:
it's a little strange. first: no, nuget isn't the only package manager for C# or .NET generally. But inside FOSSA CLI we group all the tactics for .NET under the `Nuget` module, e.g. `Strategy.NuGet.Paket`.

i do think it probably makes more sense to refer to the package manager in this table like we do for other languages.

Reply:
I've gone ahead and updated these to match the other package managers in the table: C# was the only odd one out.

Reply:
I think that looks good, thanks!